hnn_core/tests/test_drives.py

```diff
@@ -214,10 +214,6 @@ def test_add_drives():
                        match='Duration of Poisson drive cannot be negative'):
         net.add_poisson_drive('poisson1', tstart=10, tstop=1,
                               location='distal', rate_constant=10.)
-    with pytest.raises(ValueError,
-                       match='End time of Poisson drive cannot exceed'):
-        net.add_poisson_drive('poisson1', tstop=params['tstop'] + 1,
-                              location='distal', rate_constant=10.)
```

Comment:
are we okay with removing these tests? I think we agreed that we would simply truncate the drives to be until the simulation end time, correct? This would also have the benefit that the user will not have to manually edit the `tstop` in each of the drives defined (either through API or using `add_drives_from_params`) when the simulation end time is reduced ...